core/modules/node/js/node.types.js

```diff
@@ -99,6 +99,12 @@ Backdrop.behaviors.contentTypes = {
       vals.push(Backdrop.checkPlain($(context).find('input[name="path_pattern"]').val()) || Backdrop.t('No URL pattern set'));
       return vals.join(', ');
     });
+
+    // Focus the input#edit-path-pattern field when clicking on the token
+    // browser on the /admin/structure/types/add pages (add a content type).
+    $('#edit-path > div.fieldset-wrapper a').click(function(){
+      $('input#edit-path-pattern').focus();
```

Comment:
This looks good to add this functionality here but a few things we could improve:

- Let's add a class to the token link so the selector has less chance of targeting a different link (if one were added through hook_form_alter()).
- Rather than searching the whole page, the first line should be: `$context.find('#edit-path .token-browser-link')`, using $context.find(). Otherwise we'll end up double-binding this event after every AJAX requests.